news/4094.bug

```diff
@@ -0,0 +1 @@
+Allow excluding automatic updates for specific releases to operate on bugs
```

Comment:
I feel like this could be more descriptive.

Maybe:
Allow bodhi to not operate automatically on bugs linked to in changelog for specific releases

Reply:
:+1: much better, I'm going to change the message